### ISSUE TITLE
Can I use gifted chat with any XMPP Framework or Firebase? How !

### ISSUE THREAD

**Issue by asharamseervi** [FaridSafi/react-native-gifted-chat#1039]
Hello Community !!
This is same silly how-to question, you deal often, but I explored first, and nowhere mentioned such things. Can you point me to correct doc? Question is:

Can i use Gifted Chat any XMPP Frameworks like Openfire/eJabber/MongooseIM or any other framework? I saw some queries related Firebase, but i hadn't got any docs, how to add integrate these? I can I add backend?

**Comment by asharamseervi** [comment]
Hello @FaridSafi, Any updates?

**Comment by brortao** [comment]
Yes. You'll need to implement an XMPP client -- for an example, see e.g. [converse.js](https://github.com/conversejs/converse.js). It uses [strophe.js](http://strophe.im/strophejs/), which is probably the most mature XMPP library for Javascript. You can then use gifted chat to display the messages as you like.

It shouldn't matter which XMPP server you use for the backend.
